Makefile

```diff
@@ -1032,13 +1033,21 @@ ifeq ($(XZ), 0)
 endif
 
 .PHONY: bench-all
-bench-all:
+bench-all: bench-addons
 	@echo "Please use benchmark/run.js or benchmark/compare.js to run the benchmarks."
 
 .PHONY: bench
-bench:
+bench: bench-addons
 	@echo "Please use benchmark/run.js or benchmark/compare.js to run the benchmarks."
 
+# Build required addons for benchmark before running it.
+.PHONY: bench-addons
+bench-addons: benchmark/misc/function_call/build/Release/binding.node
```

Comment:
This was previously the `bench-misc` target, which was removed along with most of the `bench-*` targets in https://github.com/nodejs/node/pull/18150.

Ref: https://github.com/nodejs/node/issues/17053
cc @joyeecheung @nodejs/benchmarking

Reply:
The previous plan was to invoke node-gyp inside the benchmark, but I couldn't make the child process arguments work as intended.

Reply:
Thanks for the info... so what's the preferred way? Are we taking this approach to do it in `Makefile`, or using `node-gyp` to make it be able to run under `benchmark` dir? I'm neutral to either one.